docs/conventions.md

```diff
@@ -0,0 +1,27 @@
+# Data Supply Conventions
+
+## Metadata
+_Data supply_ will associates certain files together. If it finds a CSV it will look for a corespondingly named YAML file to provide infomration about that data.
```

Comment:
```suggestion
_Data supply_ will associate certain files together. If it finds a source data file, it will look for a correspondingly named metadata file to provide information about that data.
```